src/Donations/index.tsx

```diff
@@ -19,6 +19,21 @@ import {
 } from "src/Utils/donationStepConstants";
 import { useRouter } from "next/router";
 import BackButton from "public/assets/icons/BackButton";
+import VerifiedIcon from "@mui/icons-material/Verified";
+import Popover from "@material-ui/core/Popover";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles, createStyles, Theme } from "@material-ui/core/styles";
```

Comment:
Please use the current mui dependencies. You will need to update the code, and also check that functionality/UX is as expected.